### ISSUE TITLE
QA for protest-related documents

### ISSUE THREAD

**Issue by InaraFernandez** [cityofaustin/techstack#5055]
The issue: 

We now have a specific document collection for all [formal complaints relating to the 2020 protests](https://alpha.austin.gov/en/police-oversight/about-community-police-review-commission/). These formal complaints should belong to three different document collections: 
- [Formal complaint documents](https://alpha.austin.gov/police-oversight/formal-complaint-documents/#page=1)
- [Formal complaints relating to the 2020 protests](https://alpha.austin.gov/police-oversight/official-complaint-documents-related-to-protests/#page=1)
- The existing "official complaints and discipline documents" document collection that is pretty much a catch-all for OPO documents that aren't reports and recommendations 

As we are unbatching the batched protest documents for the month of June, we are manually assigning them to the correct collections. The issue is with one-off non-batched protest complaints that have been trickling in since June. For these, I was writing in the "note" section of each document page the word "protest" as a tag so that they could be sorted into the protest-related doc collection. Since several people have been working on these since June, I don't think every protest complaint is appropriately tagged. 

I'd like to set up a QA process to search for protest-related terms within the summaries of each document page in order to catch and assign any rogue complaints. Would this be possible to do if I were to give you a list of terms to search for? 

Terms include:
- demonstrations
- demonstrators
-protest
-protesters 
-protestors 
- rubber bullets
- projectiles 
- tear gas
This is a preliminary list that I intend to finalize after I comb through the official doc spreadsheet that we share with OPO (our source of truth). 

Another ask is to be able preview all of the complaints matching the search before pushing to production just in case there are a couple of complaints that shouldn't be there. 
 cc: @toribr

**Comment by chiaberry** [comment]
Yeah, I can definitely search the summaries and make sure all the documents with terms in your list make it on the correct collection. Assign this to me when you're ready!
